kratos/utilities/binbased_fast_point_locator.h

```diff
@@ -145,7 +172,7 @@ class BinBasedFastPointLocator
      * @brief Function to construct or update the search database
      * @param CellSize The current size of the cell used for search
      */
-    void UpdateSearchDatabaseAssignedSize(double CellSize)
+    void UpdateSearchDatabaseAssignedSize(const double CellSize)
```

Comment:
To be consistent with this @roigcarlo

Reply:
Well that method is also wrong :S

Reply:
O dear... I think every time that is used, its wrong....

Reply:
¯\_(ツ)_/¯

Reply:
Most likely its a copy paste mistake because the nodes in element locator uses a StaticBins inside, which uses the Bucket size simple double.

Should I fix it myself or can you change it?

Reply:
Feel free to modify